### DISCUSSION TITLE
Does this work on any Daikin unit with an S21 port?

### DISCUSSION THREAD

**Discussion by syphernl** [revk/xESP32-Faikout#259] [Q&A]
My installer quoted a setup with CTXF-C units (Sensira). None of these with the `C`-prefix are on the supported list for Faikin but a few ones with `F` is listed. 
I couldn't find much about this specific type as Googling it results in "Did you mean FTXF-C?". This unit is part of the `*TXF` range where `*` can be A = Twin-Split, F= Single-split & C = Multi-Split.

I found a Technical Data sheet with a bunch of information and a connection diagram and there is a 5-pin S21 connector on there (on the bottom on PCB3)

![image](https://github.com/revk/ESP32-Faikin/assets/639906/133c221b-ecf8-486b-b960-1abebe601afd)

These units come with preinstalled WiFi (via Onecta app, likely the `BRP069B45` module) and it seems that there is a S21 port according to the connection diagram above. 
Does this mean a Faikin board could be connected to it?

I assume it should work seeing this one is part of the ATXF/CTXF/FTXF series and the latter has 3 devices on the supported list

**Comment by revk** [comment]
No idea personally, but we have quite a community here that may know an answer. Having an S21 is a good start!

**Comment by MUN0X** [comment]
I'm not familiar with that unit. But based on the fact that it comes with built-in wifi and onecta I would assume that it comes with the `BRP069C4x` module.
The daikin documentation shows that it is compatible with the wired BRC073 controller. [link](https://www.daikin.eu/en_us/products/product.table.html/CTXF-C.html)
This wired controller connects via S21, so I would assume Faikin would work as well. Some people in this community are trying to reverse S21 based on BRC073 so maybe they can confirm this. Just search for BRC073...
